app/controllers/api/students_controller.rb

```diff
@@ -16,9 +16,12 @@ def index
     if params[:order]
       students = students.order params[:order]
     end
-    render json: students,
-                 serializer: PaginatedSerializer,
-                 each_serializer: StudentIndexSerializer
+    respond_to do |format|
+      format.csv { send_data students.to_csv }
```

Comment:
The `students.to_csv` here currently does not do what it's supposed to, that's for @anthonyhbh to fix.
Once you restart your rails application, with `rake db:reset` and reset server, you can go to `localhost:3000/api/students.csv` and see what happens.